contracts/OnChainPricingMainnet.sol

```diff
@@ -162,7 +162,8 @@ contract OnChainPricingMainnet {
     uint256 private constant SECONDS_PER_DAY = 86400;
 
     // NOTE: This is effectively max loss for oracle covered swaps
-    uint256 public feed_tolerance = 500; // 5% Initially
+    // WP-L2 from watchpug for V4, lossing system-wide slippage from oracle feed
```

Comment:
Disagree with adding the extra check, am fine with the risk and would rather revert